src/Arcus.Templates.ServiceBus.Topic/Program.cs

```diff
@@ -4,19 +4,17 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-#if ExcludeSerilog
-#else
+using Serilog.Configuration;
```

Comment:
This should be in the `#if (ExcludeSerilog == false)` check, no?

Reply:
Yes, saw it just now too.

Reply:
Done that.